superbench/monitor/monitor.py

```diff
@@ -215,13 +238,21 @@ def __get_process_cpu_ticks(self):
         system_time = 0
         try:
             with open(self._cpu_file, 'r') as f:
-                for line in f:
-                    items = line.split()
-                    if items[0] == 'user':
-                        user_time = int(items[1])
-                    elif items[1] == 'system':
-                        system_time = int(items[1])
-                return user_time + system_time
+                if self.__cgroup == 1:
+                    for line in f:
+                        items = line.split()
+                        if items[0] == 'user':
+                            user_time = int(items[1])
+                        elif items[1] == 'system':
+                            system_time = int(items[1])
+                else:
+                    for line in f:
+                        items = line.split()
+                        if items[0] == 'user_usec':
+                            user_time = int(items[1]) / 10000
```

Comment:
what's the unit of the metric here? usec is 10^-6 sec?

Reply:
Not sure about the unit, but `/10000` can align with the v1's value which can also be got from `/proc/stat`

Reply:
int and float will not affect the following calculation.